src/index.jsx

```diff
@@ -1,15 +1,19 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
 
 import AssetsPage from './AssetsPage';
 import BackendStatusBanner from './BackendStatusBanner';
+import store from './data/store';
 
 const App = () => (
-  <div>
-    <BackendStatusBanner />
-    <h1>I am a app</h1>
-    <AssetsPage />
-  </div>
+  <Provider store={store}>
```

Comment:
Finally, the last step. In order to make all the data in the store accessible to the rest of the application, we need to wrap it in a `react-redux` Provider at the root level. This isn't really something we'll have to worry about ever again.